### PR TITLE
docs(derek): add pattern event tracking doc

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-9073

Visible here:
http://localhost:3000/derek/solutions/solutions-usage

Adding documentation around what patterns have events within zoolander and what those events are. This is everything that we found in discovery with data.

### CODE REVIEW

**Comment by runrog** [comment]
+1 super amazing HTML work

**Comment by amberRrucker** [comment]
@runrog if you could hold off on +1 until the WIP label is gone that would be cool too. Whatever you wanna do.

File: styleguide/derek/pattern-components/ctas/cta-bottom.ejs [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="cta-row">
       <div class="cta-buttonWrapper">
-        <a class="cta-button">Talk to a Specialist</a>
+        <a class="cta-button"  event-track>Talk to a Specialist</a>
```
Comment (bighappyface) [inline styleguide/derek/pattern-components/ctas/cta-bottom.ejs]: This needs to be class `.track-cta` - https://github.com/rackerlabs/zoolander/blob/56a2afe1eb38973f30df7a9eb07759ffee5195c5/styleguide/derek/global-components/tracking/tracking.spec.js#L219

File: styleguide/derek/pattern-components/ctas/cta.ejs [hunk 2]
```diff
@@ -13,7 +13,7 @@
     </div>
     <div class="cta-row">
       <div class="cta-buttonWrapper">
-        <a class="cta-button">Talk to a Specialist</a>
+        <a class="cta-button"  event-track>Talk to a Specialist</a>
```
Comment (bighappyface) [inline styleguide/derek/pattern-components/ctas/cta.ejs]: This needs to be class `.track-cta` - https://github.com/rackerlabs/zoolander/blob/56a2afe1eb38973f30df7a9eb07759ffee5195c5/styleguide/derek/global-components/tracking/tracking.spec.js#L219

File: styleguide/derek/pattern-components/tabs/tabs-horizontal.ejs [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <div class="container">
   <div class="rsTabs-container rsTabsHorizontal-container">
     <ul class="nav nav-tabs rsTabs rsTabsHorizontal">
-      <li class="rsTab-tab active"><a class="rsTab-link rsTabHorizontal-link" href="#tab-1" data-toggle="tab">Tab 1</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-2" data-toggle="tab">Tab 2</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-3" data-toggle="tab">Tab 3</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-4" data-toggle="tab">Tab 4</a></li>
+      <li class="rsTab-tab active"><a class="rsTab-link rsTabHorizontal-link" href="#tab-1" data-toggle="tab" event-track>Tab 1</a></li>
```
Comment (bighappyface) [inline styleguide/derek/pattern-components/tabs/tabs-horizontal.ejs]: These `event-track` attributes can be removed. Tracking is handled by JS code in www acting on the `.rsTabs` class.

File: styleguide/derek/pattern-components/tabs/tabs-horizontal.ejs [hunk 4]
```diff
@@ -2,9 +2,9 @@
   <div class="rsTabs-container rsTabsHorizontal-container">
     <ul class="nav nav-tabs rsTabs rsTabsHorizontal">
       <li class="rsTab-tab active"><a class="rsTab-link rsTabHorizontal-link" href="#tab-1" data-toggle="tab">Tab 1</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-2" data-toggle="tab">Tab 2</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-3" data-toggle="tab">Tab 3</a></li>
-      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-4" data-toggle="tab">Tab 4</a></li>
+      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-2" data-toggle="tab" event-track>Tab 2</a></li>
+      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-3" data-toggle="tab" event-track>Tab 3</a></li>
+      <li class="rsTab-tab"><a class="rsTab-link rsTabHorizontal-link" href="#tab-4" data-toggle="tab" event-track>Tab 4</a></li>
```
Comment (WidgetsBurritos) [inline styleguide/derek/pattern-components/tabs/tabs-horizontal.ejs]: Can you take these `event-track` attributes off?  They are not necessary.

**Comment by amberRrucker** [comment]
@WidgetsBurritos, missed those. sorry! it's updated now.

**Comment by pobtastic** [comment]
@amberRrucker Hiya! Please can you rebase master here, so we can get this show on the road 😉

**Pull request merged**